crawler/typing.py

```diff
@@ -2,5 +2,5 @@
 from typing import Callable, Optional, Union
 
 
-FieldValue = Optional[Union[str, date]]
-Cast = Callable[[str], Optional[Union[str, date]]]
+FieldValue = Optional[Union[date, int, str]]
+Cast = Callable[[str], Optional[Union[str, date, int]]]
```

Comment:
Lendo seui comentário, fiquei pensando… será que isso funciona, para evitar repetição?

```suggestion
FieldValue = Optional[Union[date, int, str]]
Cast = Callable[[str], FieldValue]
```

Reply:
Deu certo. atualizei

Reply:
Coisa linda :purple_heart: